analysis/warriorfury/src/modules/shadowlands/legendaries/NaturesFury.tsx

```diff
@@ -0,0 +1,57 @@
+import SPELLS from 'common/SPELLS';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+class NaturesFury extends Analyzer {
+  _ancientAftershockDamage: number = 0;
+  _naturesFuryDamage: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active =
```

Comment:
In all reality should the whole module be disabled if they don't have the legendary?
From what it seems this stat shows both for the covenant in general and the legendary... might want to break it up?